src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Product.xml

```diff
@@ -34,11 +34,24 @@
                     <attribute>sylius.api.product_taxon_code_filter</attribute>
                 </attribute>
             </collectionOperation>
+
             <collectionOperation name="post">
                 <attribute name="denormalization_context">
                     <attribute name="groups">product:create</attribute>
                 </attribute>
             </collectionOperation>
+
+            <collectionOperation name="get_latest">
```

Comment:
```suggestion
            <collectionOperation name="product_latest">
```

Reply:
At least should correspond to https://github.com/Sylius/Sylius/pull/11291/files#diff-2620115d9fd4902065fef0e4af355ceeR30